### DISCUSSION TITLE
Err: Expected 2D (unbatched) or 3D (batched) input to conv1d, but got input of size: [1, 32, 134, 14

### DISCUSSION THREAD

**Discussion by aptx1231** [PaddlePaddle/PaddleSpatial#189] [BaiduKDDCup2022] [answered]
Is it possible to provide more detailed information on error reporting? We have no problems when testing locally with `evaluation.py`. Our team name is BUAA_BIGSCity and the commit time is 2022-06-18 13:30 and 2022-06-18 13:55. Thank you.

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

What we can see in the log is similar to what you saw:
//===========

Imported modules in main.py
RuntimeError: Expected 2D (unbatched) or 3D (batched) input to conv1d, but got input of size: [1, 32, 134, 145]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/aistudio/eval_main.py", line 83, in main
    ret = evaluation.eval(submit_file)
  File "/home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/pytorch/evaluation.py", line 336, in eval
    raise Exception(msg)
Exception: Err: Expected 2D (unbatched) or 3D (batched) input to conv1d, but got input of size: [1, 32, 134, 145]! (GWNET.zip)
utils
Imported modules in main.py
sys
Imported modules in main.py
loss

**Reply by aptx1231** [reply]
Thanks, we will modify our code according to this
